src/core/Track.cpp

```diff
@@ -660,6 +759,9 @@ DataFile TrackContentObjectView::createTCODataFiles(
 		tcoElement.setAttribute( "trackIndex", trackIndex );
 		tcoElement.setAttribute( "trackType", tcoTrack->type() );
 		tcoElement.setAttribute( "trackName", tcoTrack->name() );
+		tcoElement.setAttribute( "color", ( *it )->m_tco->color().name() );
+		tcoElement.setAttribute( "styleColor", ! ( *it )->m_tco->hasColor() );
+		tcoElement.setAttribute( "clipColor", ( *it )->m_tco->usesCustomClipColor() );
```

Comment:
While testing, I found out that removing this breaks the copying of clip colors for some reason. I've removed the other two, but I had to leave this because the color copying wouldn't work otherwise.

Reply:
I can't reproduce this locally - can you explain what to do?

Reply:
I tested again and it works now... I guess it had something to do with a removed piece of code. Either ways, it's now removed.